src/libraries/Microsoft.PowerFx.Core/Localization/Strings.cs

```diff
@@ -448,6 +448,7 @@ internal static class TexlStrings
         public static StringGetter AboutIndex = (b) => StringResources.Get("AboutIndex", b);
         public static StringGetter IndexArg1 = (b) => StringResources.Get("IndexArg1", b);
         public static StringGetter IndexArg2 = (b) => StringResources.Get("IndexArg2", b);
+        public static StringGetter AboutPatch = (b) => StringResources.Get("AboutPatch", b);
```

Comment:
>AboutPatch [](http://example.com/codeflow?start=76&length=10)

We need the resources for the function parameters as well (like IndexArg1, IndexArg2 above)

Reply:
Is there a good way to test these prior to push?